### DISCUSSION TITLE
Template tutorial inaccurate text

### DISCUSSION THREAD

**Discussion by zjkjsd** [scikit-hep/iminuit#822] [Q&A] [answered]
Aloha iminuit community,

Thank you for making the iminuit available, it is a very handy tool! I am learning it through the [tutorials](https://iminuit.readthedocs.io/en/stable/notebooks/template_fits.html) and am particularly interested in template fitting. I noticed that the first half of the tutorial mentions that there is no built-in cost function for a template fit but then `iminuit.cost.Template` is introduced in the second half, which confused me. Maybe, I should interpret it as
```
there is no built-in cost function for a template fit with Barlow and Beeston (BB) method, 
but a built-in cost function with BB-lite methods is provided.
```

Is this correct? Sorry if this question is too simple or answered elsewhere.

Mahalo,
Boyang

**Comment by HDembinski** [comment, marked as answer]
Thank you for the kind words on iminuit and thank you very much for that pointer. The example evolved from a state where we did not have a builtin Template class and this is a leftover. I will fix this. If you find issues like this, please do not hesitate to write an issue.
